### PR TITLE
Add cipher and hash options to luks_create

### PR DESCRIPTION
**Pull request by Arnoways**
##### SUMMARY
Pull request to Add the "cipher" and "hash" option to luks_create, allowing the user to specify the encryption they want.

##### ISSUE TYPE
- Feature Pull Request

##### COMPONENT NAME
luks_device

##### ADDITIONAL INFORMATION
These options will only be used at LUKS container creation, since they are coming from luksFormat in cryptsetup.
I refered to the manpage to write the documentation, I hope it will fit into the module.

First PR here, I'm obviously open to critics, suggestions etc... :smiley:

### CODE REVIEW

File: plugins/modules/luks_device.py [hunk 1]
```diff
@@ -152,6 +152,22 @@
...
+            - "This option allows the user to define the cipher specification
+              string for the LUKS container."
+            - "Will only be used on container creation."
+            - "For pre-2.6.10 kernels, use "aes-plain" as they don't understand 
+              the new cipher spec strings. To use ESSIV, use 'aes-cbc-essiv:sha256'." 
+        type: str
+        version_added: '1.1.0'
+    hash:
+        description:
+            - "This option allows the user to specify hash used in LUKS
```
Comment (felixfontein) [inline plugins/modules/luks_device.py:166]:
```suggestion
            - "This option allows the user to specify the hash function used in LUKS
```

Comment (Arnoways) [reply plugins/modules/luks_device.py]: Thank you! Updated.

**Comment by felixfontein** [comment]
CC @mulatinho you might also be interested in this.

**Comment by felixfontein** [comment]
@Arnoways thanks a lot for contributing this!

**Pull request merged**